cost-analyzer/values.yaml

```diff
@@ -3272,6 +3272,21 @@ costEventsAudit:
 #     key: ""
 #     secretname: productkeysecret # Reference an existing k8s secret created from a file named productkey.json of format { "key": "enterprise-key-here" }. If the secretname is specified, a configmap with the key will not be created.
 #     mountPath: "/some/custom/path/productkey.json" # (use instead of secretname) Declare the path at which the product key file is mounted (eg. by a secrets provisioner). The file must be of format { "key": "enterprise-key-here" }.
+#   # The following block enables the use of a custom SMTP server which overrides Kubecost's built-in, external SMTP server for alerts and reports
+#   smtp:
+#     enabled: false
+#     config:  |
+#       {
+#         "sender_email": "",
+#         "host": "",
+#         "port": 587,
+#         "authentication": true,
+#         "username": "",
+#         "password": "",
+#         "secure": true
```

Comment:
Has a custom cert as an input value been implemented in the code?

Reply:
[It has indeed](https://github.com/kubecost/kubecost-cost-model/blob/f1ff0fafec709d9de6d7a5c55f79548bacb26525/pkg/messaging/smtp/config.go#L15).

Reply:
It is an allowable value, but certificates have not yet been tested. Given the big push to get this into nightly, I'll be focusing on certificates once everyone has access to this feature in nightly.